test/functional/mempool_limit.py

```diff
@@ -76,6 +83,44 @@ def run_test(self):
         self.log.info('Create a mempool tx that will not pass mempoolminfee')
         assert_raises_rpc_error(-26, "mempool min fee not met", miniwallet.send_self_transfer, from_node=node, fee_rate=relayfee)
 
+        self.log.info("Check that submitpackage allows cpfp of a parent below mempool min feerate")
+        node = self.nodes[0]
+        peer = node.add_p2p_connection(P2PTxInvStore())
+
+        # Package with 2 parents and 1 child. One parent pays for itself using modified fees, and
+        # another has 0 fees but is bumped by child.
```

Comment:
490f3c1a8eabac737e196e7fffd7765aa941f65a
I assume another parent is `tx_poor`, meaning it pays `relayfee` rather than 0?

Reply:
Thanks, I've updated this comment along with a few others.